constants.py

```diff
@@ -29,9 +29,12 @@
 NS_POLICY_ANNOTATION = "net.beta.kubernetes.io/network-policy"
 
 # Tier name /order to use for policies.
-NET_POL_TIER_NAME = "k8s-network-policy"
+NET_POL_TIER_DEFAULT = "default"
```

Comment:
So, I know we've got a bunch of constants in here right now, but I think this would be a good opportunity to remove this one and just embed "default" directly into the code.  I like that better since it's one less indirection you need to take to see what is happening when reading the code.

WDYT?